src/content/docs/fr/reference/errors/unsupported-config-transform-error.mdx

```diff
@@ -0,0 +1,15 @@
+---
+titre : Transformation non prise en charge dans la configuration du contenu. 
+i18nPrêt : true 
+githubURL : https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
```

Comment:
```suggestion
title: Transformation non prise en charge dans la configuration du contenu. 
i18nReady: true 
githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
```

Reply:
Good call! I was fixing these at the same time! 😄